src/services/extra_data.py

```diff
@@ -57,15 +57,21 @@ def __init__(self, w3: Web3):
         self.w3 = w3
 
     def collect(self, blockstamp: BlockStamp, exited_validators: list[LidoKey], stucked_validators: list[LidoKey]) -> bytes:
```

Comment:
Here I made a mistake, most likely we will return here not keys but lists of (module, op_id, keys).
We shouldn't report values if they wasn't changed, so we need to compare them. - Or we could add this functionality here?

Reply:
Yes, we could. Share with me some knowledge about this